spec/features/timecards_spec.rb

```diff
@@ -62,4 +62,49 @@
       expect(page).to have_content("Home")
     end
   end
+
+  context "Manager Usage" do
+    before(:each) do
+      sign_in_as('Manager')
+    end
+
+    it "check if 'Verify' button exists on all applicable pages" do
+      @tc = create(:timecard, person: @cj, status: 'Unverified')
+
+      visit timecards_path
+      expect(page).to have_button('Verify')
+
+      visit person_path(@cj)
+      expect(page).to have_button('Verify')
+
+      visit timecard_path(@tc)
+      expect(page).to have_button('Verify')
+    end
+
+    it "click 'Verify' button for persons timecard from all applicable pages" do
+      @tc1 = create(:timecard, person: @cj, status: 'Unverified')
+
+      visit timecards_path
+      click_button "Verify"
+
+      expect(page).not_to have_button('Verified')
+      expect(page).not_to have_content('Unverified')
+
+      @tc2 = create(:timecard, person: @cj, status: 'Unverified')
```

Comment:
I really like the number of tests, but  I think it would be better to break this into three blocks. You could wrap it in a context.